src/main/java/Model/Basket.java

```diff
@@ -0,0 +1,8 @@
+package Model;
+
+import java.util.ArrayList;
+
+public class Basket {
+    Float summaryValue;
+    ArrayList<Product> basket = new ArrayList<>();
```

Comment:
Don't initialize variable here. It should be done eg. in the constructor.

Reply:
Done.